### PR TITLE
Updated Catalog Objects log messages

### LINKED ISSUE #836: [BUG]: Recon Installation failing without any error message
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Category of Bug / Issue

ReconcileError

### Current Behavior

Installation fails with error - 
```
Aborting the installation.
Error: installer: exit status 1
```

### Expected Behavior

Installation failing should tell the probable cause of the error.

### Steps To Reproduce

If insufficient privileges are given (READ_VOLUME, WRITE_VOLUME are not provided), the installation fails without any error message

### Relevant log output or Exception details

```shell
Aborting the installation.
Error: installer: exit status 1
```

### Sample Query

_No response_

### Operating System

macOS

### Version

latest via Databricks CLI

### PR DESCRIPTION
**Pull request by vijaypavann-db**
Updated catalog, schema, and volume logging messages, when there are:
*  no object found
*  insufficient privileges

closes #836

### CODE REVIEW

**Comment by vijaypavann-db** [comment]
1. Scenario: Catalog not found. Create Catalog? no.
catalog_not_found_create_no_exit

2. Scenario: Catalog found but User doesn't have sufficient Privileges to use the Catalog. Choose another Catalog? no.
catalog_found_no_privileges_create_no_exit

- Similar log messages for schema and volume

File: src/databricks/labs/remorph/deployment/configurator.py [hunk 1]
```diff
@@ -44,15 +44,15 @@ def prompt_for_catalog_setup(
...
                 if not self._prompts.confirm("Do you want to use another catalog?"):
-                    raise SystemExit("Aborting the installation.")
+                    raise SystemExit("Please choose the correct catalog. Aborting the installation.")
             else:
                 if self._prompts.confirm(f"Catalog `{catalog_name}` doesn't exist. Create it?"):
                     result = self._catalog_ops.create_catalog(catalog_name)
                     assert result.name is not None
                     return result.name
-                raise SystemExit("Aborting the installation.")
+                raise SystemExit("Please create the catalog. Aborting the installation.")
```
Comment (sundarshankar89) [inline src/databricks/labs/remorph/deployment/configurator.py:55]:
```suggestion
                raise SystemExit("Cannot continue installation, without valid catalog, Aborting the installation.")
```

File: src/databricks/labs/remorph/deployment/configurator.py [hunk 2]
```diff
@@ -74,17 +74,17 @@ def prompt_for_schema_setup(
...
                 if not self._prompts.confirm("Do you want to use another schema?"):
-                    raise SystemExit("Aborting the installation.")
+                    raise SystemExit("Please choose the correct schema. Aborting the installation.")
             else:
                 if self._prompts.confirm(f"Schema `{schema_name}` doesn't exist in catalog `{catalog}`. Create it?"):
                     result = self._catalog_ops.create_schema(schema_name, catalog)
                     assert result.name is not None
                     return result.name
-                raise SystemExit("Aborting the installation.")
+                raise SystemExit("Please create the schema. Aborting the installation.")
```
Comment (sundarshankar89) [inline src/databricks/labs/remorph/deployment/configurator.py:87]: Same as above

File: src/databricks/labs/remorph/deployment/configurator.py (hunk 1 above)
Comment (sundarshankar89) [reply src/databricks/labs/remorph/deployment/configurator.py:55]: Why do we have to change to warning why info logs are not working?

**Review by sundarshankar89 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**